nixnet/system/system.py

```diff
@@ -68,6 +70,11 @@ def close(self):
         self._handle = None
 
     @property
+    def databases(self):
```

Comment:
It's just a type annotation! I don't think it's super critical that needs to be addressed right away, adding notes to the PR